.deployments/ropsten/ropsten-1654132332519000.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
It looks like not everything was deployed here, should we delete this file?

Reply:
Yes. 
I'm deleting all these files because they are tests.